com.github.scrivano.scrivano.yml

```
@@ -0,0 +1,31 @@
+app-id: com.github.scrivano.scrivano
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: Scrivano
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
```

```suggestion
  - --socket=fallback-x11
  - --socket=wayland
```

if it supports wayand, x11 should be obly fallback.

---

Thanks for letting me know, done.